charts/consul/templates/api-gateway-controller-clusterrolebinding.yaml

```
@@ -0,0 +1,20 @@
+{{- if .Values.apiGateway.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "consul.fullname" . }}-api-gateway-controller-role-binding
```

Hey @nickethier! I've been working on #993 and just making those updates here as well!
```suggestion
  name: {{ template "consul.fullname" . }}-api-gateway-controller
```